package.json

```diff
@@ -83,10 +84,11 @@
     "build": "npm-run-all --parallel build:js build:css --serial build:gzip size",
     "clean": "rm -rf packages/*/lib packages/@uppy/*/lib && rm -rf packages/uppy/dist",
     "lint:fix": "npm run lint -- --fix",
+    "lint:companion": "eslint --ignore-path packages/@uppy/companion/.eslintignore -c packages/@uppy/companion/.eslintrc.json packages/@uppy/companion/.",
     "lint": "eslint . --cache",
     "lint-staged": "lint-staged",
     "release": "./bin/release",
-    "start:server": "node bin/start-server",
+    "start:companion": "sh ./bin/companion",
     "start": "npm-run-all --parallel watch start:server web:preview",
```

Comment:
`start:server` → `start:companion` in the `start` script

Reply:
fixed